src/middleware/validator/user-course.validator.ts

```diff
@@ -0,0 +1,16 @@
+import { check } from 'express-validator'
+
+import { userCourseRoles } from 'devu-shared-modules'
+
+import validate from './generic.validator'
+
+// TODO: check if user/course id exist
```

Comment:
Once we add that foreign key constraint then the db won't allow it to insert unless the id exists